content/en/docs/configuration/dependencies.md

```diff
@@ -121,72 +121,69 @@ $ rebar3 update
 To use a CDN other than the default, such as one of the [official mirrors](https://hex.pm/docs/mirrors) add to your project's `rebar.config` or to `~/.config/rebar3/rebar.config`:
 
 ```erlang
-{rebar_packages_cdn, "https://s3-eu-west-1.amazonaws.com/s3-eu.hex.pm"}. 
+{rebar_packages_cdn, "https://s3-eu-west-1.amazonaws.com/s3-eu.hex.pm"}.
 ```
 
 ## Checkout Dependencies
 
 To handle the case of dependencies you wish to work on locally without having to constantly publish new versions, there is the `_checkouts` directory. Simply make a symlink or copy your dependency to `_checkouts` at the top level of your project:
 
-```
+```shell
 _checkouts
 └── depA
     └── src
 ```
 
 Any application or plugin in `_checkouts` will take precedence over the same application if it is additionally listed in the `rebar.config`'s `deps`, `plugins`, or `project_plugins`. This also overrides anything already fetched to `_build`.
 
-Note that `_checkouts` is an override, this means that for it to work a `dep` or `plugin` entry in rebar.config _must_ exist.
+Note that `_checkouts` is an override, this means that for it to work a `dep` or `plugin` entry in `rebar.config` _must_ exist.
 
 ## Fetching Order
 
 For a regular dependency tree, such as:
 
-```
+```shell
```

Comment:
Is this something the linter demanded? these are absolutely not shell sections, they're ascii art. There's no reason they would need syntax highlighting :/

Reply:
Yeah, linter. I might remove those, sure. Not sure it's being highlighted, though. Could use ```plain, for example.

Reply:
yeah that'd be fine.

Reply:
511cdb9